### ISSUE TITLE
[Bug] Null Pointer exception while showdown importing some pokemon.

### ISSUE THREAD

**Issue by architdate** [kwsch/PKHeX#1908]
In `CommonEdits.cs` file, ApplySetDetails function, the current implementation is such that `legal.Info` can be null so `legal.Info.Relearn` will give a Null Pointer exception. One of the ways to reproduce this can be: In `Debug` mode build the application, load up an Omega Ruby save and try to import the following:
```
Giratina-Origin @ Griseous Orb
Ability: Levitate
EVs: 252 Def / 244 SpA / 12 Spe
Modest Nature
- Hex
- Draco Meteor
- Thunder Wave
- Defog
```
The following error can be viewed:
![image](https://user-images.githubusercontent.com/17220860/39098536-e3dfed66-469e-11e8-89c0-01b0751f933d.png)
A possible implementation could be to try except it,
Also if possible sets like these could also have suggested encounters set as well :) 
As always keep up the great work!

**Comment by kwsch** [comment]
Duplicate: https://github.com/kwsch/PKHeX/issues/1904
double check you're using latest commit:
https://github.com/kwsch/PKHeX/blob/2ed003d6685b66a7eb58f8e6c8cf79bc8812afb7/PKHeX.Core/PKM/Editing/CommonEdits.cs#L331-L334

**Comment by architdate** [comment]
Oh, will check and close if resolved! Thanks for the quick response

**Comment by architdate** [comment]
Confirmed. Resolved in #1904. Sorry for the duplicate issue request
